aws_marketplace/using_data/using_data_with_ml_model/using-dataset-product-from-aws-data-exchange-with-ml-model-from-aws-marketplace.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>You should still give instructions to clean up all resources in this notebook even if they do not lead to charges. Please specify all resources that should be deleted, including model, endpoint config, and any others.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
The notebook relies upon the user to use native marketplace cloudformation template to set up the endpoint and all other resources. The cleanup section advices on user to terminate the template.